subscriptions.go

```diff
@@ -191,6 +192,9 @@ type UpdateSubscription struct {
 	NetTerms               NullInt              `xml:"net_terms,omitempty"`
 	PONumber               string               `xml:"po_number,omitempty"`
 	SubscriptionAddOns     *[]SubscriptionAddOn `xml:"subscription_add_ons>subscription_add_on,omitempty"`
+	GatewayCode            string               `xml:"gateway_code,omitempty"`
+	CustomerNotes          string               `xml:"customer_notes,omitempty"`
```

Comment:
I don't see either of these fields documented here:
https://dev.recurly.com/docs/update-subscription

You can update notes with this API call:
https://dev.recurly.com/docs/update-subscription-notes

We also have a `SubscriptionNotes` struct. Could you provide more info?

Reply:
I removed `CustomerNotes` as I am not sure if we can update it in regular subscription update API. i will propose a separate change for it as needed